<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>642</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>642</width>
     <height>422</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>642</width>
     <height>422</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>642</width>
      <height>422</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>642</width>
      <height>422</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>642</width>
      <height>422</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="StartMenu">
     <property name="minimumSize">
      <size>
       <width>642</width>
       <height>422</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>642</width>
       <height>422</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>640</width>
       <height>420</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QGraphicsView" name="startBackView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>642</width>
        <height>422</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>642</width>
        <height>422</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>642</width>
        <height>422</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>420</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="New_game">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>NEW GAME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Connect">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>CONNECT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Placing">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>642</width>
       <height>422</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>642</width>
       <height>422</height>
      </size>
     </property>
     <widget class="QGraphicsView" name="placingBackVIew">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>642</width>
        <height>422</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>642</width>
        <height>422</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>642</width>
        <height>422</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Game">
     <property name="minimumSize">
      <size>
       <width>642</width>
       <height>422</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>642</width>
       <height>422</height>
      </size>
     </property>
     <widget class="QGraphicsView" name="fieldBackView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>642</width>
        <height>422</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>642</width>
        <height>422</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>642</width>
        <height>422</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
